### PR TITLE
EZP-32361: Fixed Subtree copying when child is invisible

### PR DESCRIPTION
**Pull request by barw4**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [EZP-32361](https://jira.ez.no/browse/EZP-32361)
| **Bug**| yes
| **New feature**    | no
| **Target version** | `7.5`
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**    | no

**TODO**:
- [x] Fix a bug.
- [x] Implement tests.
- [x] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [x] Ask for Code Review.

### CODE REVIEW

**Comment by adamwojs** [comment]
@barw4 CI is failing here.

**Review by alongosz — APPROVED** [review]

**Review by adamwojs — APPROVED** [review]

File: eZ/Publish/API/Repository/Tests/LocationServiceTest.php [hunk 1]
```diff
@@ -2520,6 +2520,49 @@ public function testCopySubtreeIncrementsChildCountOfNewParent()
...
+
+        // Load Subtree properties after copy
+        $actual = $this->loadSubtreeProperties($copiedLocation);
+
+        self::assertEquals(count($expected), count($actual));
+
+        foreach ($actual as $key => $properties) {
+            self::assertEquals($expected[$key]['hidden'], $properties['hidden']);
+            self::assertEquals($expected[$key]['invisible'], $properties['invisible']);
+        }
```
Comment (Steveb-p) [inline eZ/Publish/API/Repository/Tests/LocationServiceTest.php:2563]: In general usage of `self::assertSame` is preferred. It's similar to using `==` vs `===` - if there is no reason for non-strict comparison, strict should be used.

**Review by Steveb-p — APPROVED** [review]

**Review by kacper-wieczorek-ibexa — APPROVED** [review]
QA Approve. Tested on v2.5.19 ee, v3.2.7-dev experience, 3.3.3-dev experience.

**Comment by lserwatka** [comment]
You can merge it up.

**Comment by barw4** [comment]
Merged into:
`1.2`: https://github.com/ezsystems/ezplatform-kernel/commit/1a8e058b1f4ad6ce7682c19a37c7c0e89d05301a
`1.3`: https://github.com/ezsystems/ezplatform-kernel/commit/2b485679aa993cf6619aac6a49d13e300711f442
`master`: https://github.com/ezsystems/ezplatform-kernel/commit/5bf964d1ce74366432242a42d0f23596f2afa03a

**Pull request merged**